stdlib/list.mli

```diff
@@ -57,6 +57,12 @@ val compare_length_with : 'a list -> int -> int
    @since 4.05.0
  *)
 
+val is_empty : 'a list -> bool
+(** [is_empty l] is true if and only if [l] has no elements. It is equivalent to
```

Comment:
Question on style: shouldn't it be `[true]` rather than `true`? (this is definitely nitpicking :grin: And maybe the syntax you chose is the convention in Stdlib?

Reply:
```suggestion
(** [is_empty l] is true if and only if [l] has no element. It is equivalent to
```

Reply:
> Question on style: shouldn't it be [true] rather than true?

I was copying `List.mem` in style here, but have no strong opinion either way.

Reply:
Fair enough, feel free to resolve then :grin: (I can't for some reason)